src/resource/resource.ts

```diff
@@ -181,12 +181,14 @@ export function isPodOwner(
  * @since 1.2.0
  */
 export class FetchError extends SolidClientError {
+  public readonly response: Response & { ok: false };
   public readonly statusCode: number;
   public readonly statusText?: string;
```

Comment:
Isn't it redundant to have both ? should we consider making `statusCode` and `statusText` into accessors for the `response` properties for retro-compatibility, and eventually deprecate these ?

Reply:
Yes, they should be deprecated -I just didn't want to create a breaking change with this. I can swap them to getters, though.